Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fffft"
-version = "0.4.1"
+version = "0.4.2"
```

Comment:
I know I messed this up in the past, but I *think* this should be 0.5.0 rather than 0.4.2 because bumping the `ff` version breaks semver compatibility with other crates.

(For example, wouldn't this bump break `lcpc-ligero-pc`?)